src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1149,13 +1165,8 @@ private void VisitObjectElementInitializer(Symbol containingSymbol, int containi
 
         private new void VisitCollectionElementInitializer(BoundCollectionElementInitializer node)
         {
-            if (node.AddMethod.CallsAreOmitted(node.SyntaxTree))
-            {
-                // PROTOTYPE(NullableReferenceTypes): Should skip state set in arguments
-                // of omitted call. See PreciseAbstractFlowPass.VisitCollectionElementInitializer.
-            }
-
-            VisitArguments(node, node.Arguments, default(ImmutableArray<RefKind>), node.AddMethod, node.ArgsToParamsOpt, node.Expanded);
+            // Note: we analyze even omitted calls
```

Comment:
>// Note: we analyze even omitted calls [](start = 12, length = 38)

It would be good to add a test to make sure it stays this way.

Reply:
Added `OmittedInitializerCall`. Thanks